stress_test/loadtest.py

```diff
@@ -0,0 +1,90 @@
+from uuid import uuid4 as uuid
+import json
+import random
+import asyncio
+
+from molotov import scenario
+
+_SHEPHERD_URL = 'http://0.0.0.0:5000'
+
+
+def create_payload(i: int, job_id: str):
+    return {
+        'job_id': job_id,
+        'payload': json.dumps(dict(
+            sleep=str(i % 5),
+            input='42'
+        )),
+        'model': {'name': 'stress_test', 'version': ''}
+    }
+
+
+async def job_lifecycle(session):
+    job_id = str(uuid())
+    i = random.randint(1, 5)
+    json_dict = create_payload(i, job_id)
+
+    async with session.post(f'{_SHEPHERD_URL}/start-job', json=json_dict) as resp:
+        assert resp.status == 200
+
+    async with session.get(f'{_SHEPHERD_URL}/jobs/{job_id}/status') as resp:
+        assert resp.status == 200
+
+    while True:
+        async with session.get(f'{_SHEPHERD_URL}/jobs/{job_id}/result') as resp:
+            if resp.status == 200:
+                response = await resp.text()
+                response = json.loads(response)
+                assert response['result'] == '42'
+                break
+            elif resp.status != 202:
```

Comment:
since we use `assert` all over this script, maybe we could also replace this with `assert resp.status == 202`